src/GeographicalFeatures/CriticalFacilities/DetailsView/Body.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+/**
+ * @function
+ * @name CriticalFacilityDetailsViewBody
+ * @description details of a critical facility
+ * @returns {object} React component
+ * @version 0.1.0
+ * @since 0.1.0
+ */
+const CriticalFacilityDetailsViewBody = () => {
+  return <div>This is body</div>;
```

Comment:
Is this still `Work in Progress`?

Reply:
Yes. I wanted to give you a chance to look at my direction, also I did not want to send a huge PR with  many files changed